### PR TITLE
optimize api doc i18n

### PR DESCRIPTION
**Pull request by SantyWang**
Re: #

Changelog:
 *

### CODE REVIEW

File: cocos/physics/framework/components/colliders/collider.ts [hunk 1]
```diff
@@ -309,7 +307,7 @@ export class Collider extends Eventify(Component) {
      * Gets the group value.
      * @zh
      * 获取分组值。
-     * @returns 整数，范围为 2 的 0 次方 到 2 的 31 次方
+     * @returns @zh 整数，范围为 2 的 0 次方 到 2 的 31 次方 @en integer, The range is from 2^0 to 2^31
```
Comment (minggo) [inline cocos/physics/framework/components/colliders/collider.ts]: integer -> group number, it is an integer, the range is [2^0, 2^31].

File: cocos/physics/framework/physics-system.ts [hunk 2]
```diff
@@ -405,11 +403,12 @@ export class PhysicsSystem extends System implements IWorldInitData {
      * Collision detect all collider, and record all the detected results, through PhysicsSystem.Instance.RaycastResults access to the results.
      * @zh
      * 检测所有的碰撞盒，并记录所有被检测到的结果，通过 PhysicsSystem.instance.raycastResults 访问结果。
-     * @param worldRay 世界空间下的一条射线
-     * @param mask 掩码，默认为 0xffffffff
-     * @param maxDistance 最大检测距离，默认为 10000000，目前请勿传入 Infinity 或 Number.MAX_VALUE
-     * @param queryTrigger 是否检测触发器
-     * @return boolean 表示是否有检测到碰撞盒
+     * @param worldRay @zh 世界空间下的一条射线 @en A ray in world space
+     * @param mask @zh 掩码，默认为 0xffffffff @en Mask, default is 0xffffffff
```
Comment (minggo) [inline cocos/physics/framework/physics-system.ts]: default -> default value

File: cocos/physics/framework/components/rigid-body.ts [hunk 3]
```diff
@@ -437,8 +435,8 @@ export class RigidBody extends Component {
      * Apply force to a world point. This could, for example, be a point on the Body surface.
      * @zh
      * 在世界空间中，相对于刚体的质心的某点上对刚体施加作用力。
-     * @param force - 作用力
-     * @param relativePoint - 作用点，相对于刚体的质心
+     * @param force @zh 作用力 @en the force applied
```
Comment (pandamicro) [inline cocos/physics/framework/components/rigid-body.ts]: the -> The

File: cocos/physics/framework/components/rigid-body.ts [hunk 4]
```diff
@@ -449,8 +447,8 @@ export class RigidBody extends Component {
      * Apply force to a local point. This could, for example, be a point on the Body surface.
      * @zh
      * 在本地空间中，相对于刚体的质心的某点上对刚体施加作用力。
-     * @param force - 作用力
-     * @param localPoint - 作用点
+     * @param force @zh 作用力 @en the force applied
```
Comment (pandamicro) [inline cocos/physics/framework/components/rigid-body.ts]: the -> The

File: cocos/physics/framework/components/rigid-body.ts [hunk 5]
```diff
@@ -449,8 +447,8 @@ export class RigidBody extends Component {
...
+     * @param localPoint @zh 作用点 @en The point applied the force
```
Comment (pandamicro) [inline cocos/physics/framework/components/rigid-body.ts]: The point to apply the force on

File: cocos/physics/framework/components/rigid-body.ts [hunk 6]
```diff
@@ -437,8 +435,8 @@ export class RigidBody extends Component {
      * Apply force to a world point. This could, for example, be a point on the Body surface.
      * @zh
      * 在世界空间中，相对于刚体的质心的某点上对刚体施加作用力。
-     * @param force - 作用力
-     * @param relativePoint - 作用点，相对于刚体的质心
+     * @param force @zh 作用力 @en the force applied
+     * @param relativePoint @zh 作用点，相对于刚体的质心 @en The point applied the force, relative to the center of mass of the rigid body
```
Comment (pandamicro) [inline cocos/physics/framework/components/rigid-body.ts]: The point to apply the force on

File: cocos/physics/framework/components/rigid-body.ts [hunk 7]
```diff
@@ -461,8 +459,8 @@ export class RigidBody extends Component {
      * In world space, impulse is applied to the rigid body at some point relative to the center of mass of the rigid body.
      * @zh
      * 在世界空间中，相对于刚体的质心的某点上对刚体施加冲量。
-     * @param impulse - 冲量
-     * @param relativePoint - 作用点，相对于刚体的中心点
+     * @param impulse @zh 冲量 @en the impulse applied
+     * @param relativePoint @zh 作用点，相对于刚体的中心点 @en The point applied the impulse, relative to the center of mass of the rigid body
```
Comment (pandamicro) [inline cocos/physics/framework/components/rigid-body.ts]: The point to apply the impulse

File: cocos/physics/framework/components/rigid-body.ts [hunk 8]
```diff
@@ -473,8 +471,8 @@ export class RigidBody extends Component {
      * In local space, impulse is applied to the rigid body at some point relative to the center of mass of the rigid body.
      * @zh
      * 在本地空间中，相对于刚体的质心的某点上对刚体施加冲量。
-     * @param impulse - 冲量
-     * @param localPoint - 作用点
+     * @param impulse @zh 冲量 @en the impulse applied
+     * @param localPoint @zh 作用点 @en The point applied the impulse
```
Comment (pandamicro) [inline cocos/physics/framework/components/rigid-body.ts]: The point to apply the impulse

**Review by minggo — APPROVED** [review]

**Review by pandamicro — APPROVED** [review]

**Comment by pandamicro** [comment]
https://github.com/cocos/cocos-engine/issues/10840

**Pull request merged**